cmd/kubeadm/app/phases/certs/certlist.go

```diff
@@ -151,6 +152,7 @@ func (t CertificateTree) CreateTree(ic *kubeadmapi.InitConfiguration) error {
 				continue
 			}
 			// CA key exists; just use that to create new certificates.
+			fmt.Printf("[certs] Using the existing CA certificate %q and key %q\n", filepath.Join(ic.CertificatesDir, fmt.Sprintf("%s.crt", ca.BaseName)), filepath.Join(ic.CertificatesDir, fmt.Sprintf("%s.key", ca.BaseName)))
```

Comment:
Can we just use `klog.V(1)` here?  Given that our current output already illustrates what is being done.

Reply:
What I want is a way to show that the directory here is not indeed points to a dry-run dir, either `print` or `log` is fine for me.